generatebundlefile/Makefile

```
@@ -41,24 +42,48 @@ clean: ## Clean output directory, and the built binary
 
 build: ## Build release binary.
 	mkdir -p $(REPO_ROOT)/generatebundlefile/bin
-	go1.17.5 mod tidy -compat=1.17
-	go1.17.5 build -o $(REPO_ROOT)/generatebundlefile/bin/generatebundlefile *.go
+	$(GO) mod tidy -compat=1.17
+	$(GO) build -o $(REPO_ROOT)/generatebundlefile/bin/generatebundlefile *.go
 
 build-linux:
 	[ -d bin ] || mkdir bin
 	env GOOS=linux GOARCH=amd64 $(MAKE) build
 
 run:
-	go run .
+	$(GO) run .
 
 # CI for Bundle Creation and Signing
 
 dev-release: build
 	$(REPO_ROOT)/generatebundlefile/scripts/generate_bundle.sh $(ECR_PUBLIC_URI) $(KMS_KEY)
 
-oras-install:
-	curl -LO https://github.com/oras-project/oras/releases/download/v0.12.0/oras_0.12.0_linux_amd64.tar.gz
+## ORAS Make commands
+ORAS_VERSION := "0.12.0"
+ifeq ($(OS),Windows_NT)
+        # no op
+else
+	UNAME_S := $(shell uname -s)
+	UNAME_M := $(shell uname -m)
+	ifeq ($(UNAME_S),Linux)
+		ifeq ($(UNAME_M),x86_64)
+			ORAS_OSARCH := "linux_amd64"
+		endif
+	endif
+	ifeq ($(UNAME_S),Darwin)
+		ifeq ($(UNAME_M),x86_64)
+			ORAS_OSARCH := "darwin_amd64"
+		endif
+	endif
+endif
+
+.PHONY: test-oras
+test-oras:
+	@echo oras_$(ORAS_VERSION)_$(ORAS_OSARCH).tar.gz
+
+.PHONY: oras-install
+oras-install: test-oras
```

```suggestion
oras-install: 
```
Assuming you remove the `test-oras` target.